XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -42,6 +42,32 @@
             }
           ],
           "RelatedControls": [
+            "StandardUICommand",
+            "AppBarButton",
+            "AppBarToggleButton",
+            "CommandBar"
+          ]
+        },
+        {
+          "UniqueId": "StandardUICommand",
+          "Title": "StandardUICommand",
+          "Subtitle": "A set of built-in XamlUICommands representing commonly used commands",
```

Comment:
I suspect this is a copy/paste artifact - what should the updated StandardUICommand subtitle be?

Reply:
Actually I think this one is correct. StandardUICommands are a set of XAMLUICommands that are built-in. I will make this more clear.